cirq-core/cirq/ops/state_preparation_gate_test.py

```diff
@@ -0,0 +1,79 @@
+# Copyright 2019 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import numpy as np
+import cirq
+import pytest
+
+
+def test_state_prep_gate():
+    states = np.array(
```

Comment:
For testing across multiple input values, use `@pytest.mark.parametrize` instead. This will give clearer test output on failures - specifically, it will clearly identify which case failed.

Docs: https://docs.pytest.org/en/6.2.x/parametrize.html